src/ucp/rma/flush.c

```diff
@@ -383,7 +383,8 @@ ucs_status_ptr_t ucp_ep_flush_internal(ucp_ep_h ep, unsigned req_flags,
     req->send.ep                    = ep;
     req->send.flushed_cb            = flushed_cb;
     req->send.flush.prog_id         = UCS_CALLBACKQ_ID_NULL;
-    req->send.flush.uct_flags       = UCT_FLUSH_FLAG_LOCAL;
+    req->send.flush.uct_flags       = ucp_request_param_flags(param,
+                                                              UCT_FLUSH_FLAG_LOCAL);
```

Comment:
can we take it from `worker_req` instead? It does not seem to be correct to pass UCT flags thru `UCP_request_param_t`